packages/pluggableWidgets/rich-text-web/src/utils/contants.ts

```diff
@@ -1,3 +1,13 @@
+function getBaseUrl(): string {
+    const bases = document.getElementsByTagName("base");
```

Comment:
NIT: you can use destructuring with nullish coalescing operator (`??`)
```suggestion
    const [base] = document.getElementsByTagName("base");
    return base?.href ?? "";
```